node/test/telemetrytests.ts

```diff
@@ -0,0 +1,65 @@
+/// <reference path="../typings/index.d.ts" />
+/// <reference path="../_build/task.d.ts" />
+
+import assert = require('assert');
+import * as tl from '../_build/task';
+import testutil = require('./testutil');
+
+describe('Telemetry Tests', function () {

```

Comment:
can you move this into commandtests.ts to keep the number of files down (there are only two tests in this file)